### PR TITLE
Some additional CI checks

### PR DESCRIPTION
**Pull request by ihucos**
wait for tests...

### CODE REVIEW

**Comment by fuzzylogic2000** [comment]
like https://github.com/liqd/a4-product/pull/346

**Comment by ihucos** [comment]
@fuzzylogic2000 it would be really nice to have some kind of base for the projects, we could for example have a base makefile and then we `Makefile include` it in the other makefiles and overwrite what is different

**Comment by fuzzylogic2000** [comment]
Yes! That would be nice!

File: euth/offlinephases/migrations/0006_alter_offlineevent_file_upload.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        ('euth_offlinephases', '0005_copy_data'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='offlineeventfileupload',
+            name='document',
+            field=models.FileField(upload_to='offlineevents/documents', validators=[euth.offlinephases.validators.validate_file_type_and_size]),
+        ),
+    ]
```
[the remark this answers is not in the document]
Comment (fuzzylogic2000) [reply euth/offlinephases/migrations/0006_alter_offlineevent_file_upload.py]: Weeeell, if we don't include the migration this PR fails. And the migrations are from an a4-update, so we'll have to include them at some point anyway.

[the remark this answers is not in the document]
Comment (ihucos) [reply euth/offlinephases/migrations/0006_alter_offlineevent_file_upload.py]: Ok, I see we only check for this with/since this pr.

**Pull request merged**